ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/server/LocalCommandsTest.java

```diff
@@ -187,7 +187,7 @@ public void shouldFailToWrite_error() throws IOException {
   }
 
   @Test
-  public void shouldFailToCleanup() throws IOException {
+  public void shouldNotFailToCleanup() throws IOException {
```

Comment:
Might want to simulate errors on calling LocalCommandFile.readRecords() which I think is what happened in practice.

Reply:
added